### DISCUSSION TITLE
Tone when the wake word is detected

### DISCUSSION THREAD

**Discussion by obones** [toverainc/willow#145] [Ideas]
Right now, when the wake word is properly detected, only the screen turns on.
But if I'm far away, I can't see the screen and thus have no clue if the wake word got caught up properly.
Other vocal assistants play a tune to indicate they are listening for a command, I think it would be nice to have this in Willow.

**Comment by kristiankielhofner** [comment]
We had support for this initially but we found it to be pretty annoying during test and development.

That said it would be easy enough to include an additional chime and a configuration option for using it on wake. We'll also save this for post-dynamic configuration - moving to an issue for that.

**Reply by obones** [reply]
I totally understand the issue for developers with such a thing, but I believe most end user will feel it is missing.

**Reply by kristiankielhofner** [reply]
Sorry if what I wrote was confusing.

We're currently undertaking the effort to move all of the build configuration to dynamic configuration so that Willow can be flashed once and the configuration management for all of your devices can happen from there.

If we add any functionality now we will just be making more and more configuration options in the build system, code, etc that will have to move to dynamic configuration. So with a few exceptions we're kind of in a "feature freeze" ATM. We will also have the ability to onboard/do initial flash very easily via a web browser and do over the air updates. These changes will make Willow much more usable by more people. This will be the real 1.0 release.

The tone on wake is definitely something we will implement once we look at the 1.0 release.

**Reply by hamishcunningham** [reply]
As a stopgap you could build with #229
